__tests__/app.test.js

```diff
@@ -209,3 +209,44 @@ describe('GET/api/articles', () => {
         })
     })
 })
+describe('GET/api/articles/:article_id/comments', () => {
+    test('200 status with an array of comment objects that contain appropriate properties', () => {
+        return request(app)
+        .get(`/api/articles/3/comments`)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.newComments).toBeInstanceOf(Array)
+            expect(body.newComments).toHaveLength(2)
+            body.newComments.forEach((comment) => {
+            expect(comment).toMatchObject( 
+                {
+                    comment_id: expect.any(Number),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number)
+                }
+                ) 
+            })
+        })
+    })
+    test('404 code and sends an appropriate error message when given a valid but non-existent article id', () => {
+                return request(app)
+                .get('/api/articles/999/comments')
+                .expect(404)
+                .then(({body}) => {
+                    expect(body.msg).toBe('article ID not found')
+                })
+            
+    })
+    test('200 status for a valid article id that does not have any associated comments', () => {
+        return request(app)
+        .get('/api/articles/2/comments')
+        .expect(200)
+        .then(({body}) => {
+            console.log(body, 'in test')
+            expect(body).toEqual({})
+        })
+    
+})
```

Comment:
I don't agree with the assertion here, we should still return a key of comments which will have a value of an empty array if the associated article has no comments